lib/ui/semantics.dart

```diff
@@ -228,67 +228,20 @@ class SemanticsAction {
     _kSetTextIndex: setText,
   };
 
+  /// Creates a new [SemanticsAction] from an integer `value`.
+  ///
+  /// Returns `null` if the id is not a known action.
+  static SemanticsAction? fromValue(int value) => _kActionById[value];
```

Comment:
I am not super familiar with this but could this just be `SemanticsAction(value)`?

Reply:
No, from [] maybe. Could be potentially confusing, but I need to check.